docs/configuration/dashboards.en.md

```diff
@@ -0,0 +1,76 @@
+# Dashboards
+
+Dashboards can be used to display multiple resources related to a single topic on one page. Additionally, the homepage is configured this way.
+
+## Configurations
+
+| Parameter     | Description                        | Type     |
+| ------------- | ---------------------------------- | -------- |
+| `id`          | The unique dashboard id.           | Required |
+| `name`        | The dashboard name.                | Required |
+| `description` | The dashboard description.         | Optional |
+| `icon`        | The dashboard [icon](#icons) name. | Optional |
+
+!!! example
+    The following example is a list of all available dashboards configurations:
+
+    ```yaml title="app.config.yml"
+    dashboards:
+    - id: string
+      name: string
+      description: string
+      icon: string
+      contents: [content] # (1)!
+    ```
+
+    1. The [contents](#contents) of the dashboard.
+
+## Contents
```

Comment:
I would not remove it entirely since most of it will probably stay as is. However, I've added a prominent warning to the top of the `Dashboards` page. Is that fine?

Reply:
Yes, that is fine